lib/v2/rsc/radar.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    'rsc.org': {
+        _name: 'The Royal Society of Chemistry',
+        pubs: [
+            {
+                title: 'Journal',
+                docs: 'https://docs.rsshub.app/routes/journal#the-royal-society-of-chemistry',
```

Comment:
```suggestion
                docs: 'https://docs.rsshub.app/routes/journal#royal-society-of-chemistry',
```
The slug of the title in the docs does not start with `the`.